cloud-service-providers/azure/aks/prerequisites/README.md

```diff
@@ -0,0 +1,74 @@
+# Pre-requirement
+
+The keys to create AKS for NIM is to create proper GPU nodepool.  The proper GPU nodepool should have up to date driver.  This is only achievable via a preview cli extension.
+We cannot create AKS via Azure Portal GUI and we need to create it using CLI.  We need to setup our terminal with a several cli to perform the AKS creation.  Following is the detail instructions.
```

Comment:
Avoid using pronouns like we/you in the docs if possible. I need to create a proper style guide doc for the repo, but overall we have been trying to use the impersonal tense.